weblab/experiments/tests/test_processing.py

```diff
@@ -99,13 +99,15 @@ def test_uses_existing_experiment(self, mock_post,
                                       user, model_with_version, protocol_with_version):
         model = model_with_version
         protocol = protocol_with_version
-        model_version = model.repo.latest_commit.sha
-        protocol_version = protocol.repo.latest_commit.sha
+        model_version = model.repocache.get_version(model.repo.latest_commit.sha)
```

Comment:
Given this is getting the latest commit, just doing `model_version = model.repocache.latest_version` should be fine! Similarly on the next line.

Reply:
agreed, thanks